src/test/coins_tests.cpp

```diff
@@ -634,7 +641,7 @@ static void CheckAccessCoin(CAmount base_value, CAmount cache_value, CAmount exp
 {
     SingleEntryCacheTest test(base_value, cache_value, cache_flags);
     test.cache.AccessCoin(OUTPOINT);
-    test.cache.SelfTest();
+    test.cache.SelfTest(/*sanity_check=*/false);
```

Comment:
👍 for adding sanity check in every other instance - do you think it would make sense to explain the reason why the exceptions don't pass?
> Assertion failed: (attr != 2 && attr != 4 && attr != 7), function SanityCheck, file coins.cpp, line 330.
unknown location:0: fatal error: in "coins_tests/ccoins_write": signal: SIGABRT (application abort requested)